pkg/logql/error.go

```diff
@@ -1,13 +1,20 @@
 package logql
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/prometheus/prometheus/pkg/labels"
 
 	"github.com/grafana/loki/pkg/logql/log"
 )
 
+var (
```

Comment:
Maybe a comment here saying these are used to enable `errors.Is` but they aren't ever returned from our functions (which use custom error types).